releasenotes/notes/api-eventlogs-13a7043c6cacedfe.yaml

```
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    An API (``/api/repos/<owner>/<repo>/pulls/<pull>/events``) to query the
```

```suggestion
    An API endpoint (``/api/repos/<owner>/<repo>/pulls/<pull>/events``) to query the
```

To be more precise ?